src/schema/v2/order/sharedOrderTypes.ts

```diff
@@ -288,47 +290,82 @@ const DisplayTextsType = new GraphQLObjectType<any, ResolverContext>({
+        if (!messageText) return null
+        return formatMarkdownValue(messageText, format)
+      },
+    },
   },
 })
 
 const resolveDisplayTexts = (order: OrderJSON) => {
+  const isPickup = order.fulfillment_type == "pickup"
+  let titleText = "Your order"
+  let messageParts = [""]
+
   switch (order.buyer_state) {
     case "submitted":
-      return {
-        titleText: "Great choice!",
-      }
+      titleText = "Great choice!"
+      messageParts = [
+        "Thank you! Your order is being processed.<br/>You will receive an email shortly with all the details.",
+      ]
+      break
     case "payment_failed":
-      return {
-        titleText: "Payment failed",
-      }
+      titleText = "Payment failed"
+      messageParts = [
+        "To complete your purchase, please update your payment details or provide an alternative payment method.",
+      ]
+      break
     case "processing_payment":
-      return {
-        titleText: "Your payment is processing",
-      }
+      titleText = "Your payment is processing"
+      messageParts = [
+        `Thank you for your purchase. You will be notified when the work ${
+          isPickup ? "is available for pickup" : "has shipped"
+        }.`,
+      ]
+      break
     case "processing_offline_payment":
+      titleText = "Congratulations!"
+      messageParts = [
+        "Your order has been confirmed. Thank you for your purchase.",
+      ]
+      break
     case "approved":
-      return {
-        titleText: "Congratulations!",
-      }
+      titleText = "Congratulations!"
+      messageParts = [
+        isPickup
+          ? "Thank you for your purchase. A specialist will contact you within 2 business days to coordinate pickup. You can <a href='#' data-link='contact-gallery'>contact the gallery</a> with any questions about your order."
+          : "Your order has been confirmed. Thank you for your purchase.",
+      ]
+      break
     case "shipped":
-      return {
-        titleText: "Good news, your order has shipped!",
-      }
+      titleText = "Good news, your order has shipped!"
+      messageParts = ["Your work is on its way."]
+      break
     case "completed":
-      return {
-        titleText:
-          order.fulfillment_type == "pickup"
-            ? "Your order has been picked up"
-            : "Your order has been delivered",
-      }
+      titleText = isPickup
+        ? "Your order has been picked up"
+        : "Your order has been delivered"
+      messageParts = [
+        "We hope you love your purchase! Your feedback is valuable—share any thoughts with us at orders@artsy.net.",
+      ]
+      break
     case "canceled_and_refunded":
-      return {
-        titleText: "Your order was canceled",
-      }
-    default:
-      return {
-        titleText: "Your order",
-      }
+      titleText = "Your order was canceled"
+  }
+
+  return {
+    titleText: titleText,
+    messageText: messageParts.join("<br/><br/>"),
```

Comment:
This is slightly faffy, but switches with returns are a bit nicer and more safe, since it precludes the ability to make mistakes around `break` statements (while also being a bit easier to read). 

Totally cool to just call `.join` when/where you need it (on messageParts), or define a helper